tsconfig.json

```
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "moduleResolution": "node",
+        "module": "ESNext",
+        "target": "es2022",
+        "types": [
+            "node",
+            "@wdio/globals/types",
+            "expect-webdriverio",
+            "@wdio/mocha-framework"
+        ],
+        "skipLibCheck": true,
+        "noEmit": true,
+        "allowImportingTsExtensions": true,
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "strict": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "noFallthroughCasesInSwitch": true
+    },
+    "include": [
+        "src"
+    ]
```

tsconfig is invalid, replace with the following:
```suggestion
    "include": [
        "wdio.conf.ts",
        "test"
    ]
```

---

fixed in 82c05c6f460f96c3e0d3ecf720c2b7ffaeb67bb1

---

`test` is still missing

---

fixed in fcd136bdd77d7d8e9893594c6b03fea010fcc44c